core/spec/models/spree/shipment_spec.rb

```diff
@@ -231,89 +199,7 @@
   end
 
   context "#update!" do
```

Comment:
add in `https://github.com/spree/spree/pull/4347` instead of number. change context to 1 level and name it 'transition to shipping with adjustments'